cmd/git-sync/main.go

```diff
@@ -60,6 +62,8 @@ var flRoot = flag.String("root", envString("GIT_SYNC_ROOT", envString("HOME", ""
 	"the root directory for git-sync operations, under which --dest will be created")
 var flDest = flag.String("dest", envString("GIT_SYNC_DEST", ""),
 	"the name of (a symlink to) a directory in which to check-out files under --root (defaults to the leaf dir of --repo)")
+var flError = flag.String("error", envString("GIT_SYNC_ERROR", ""),
```

Comment:
how about flErrorFile and --error-file and GIT_SYNC_ERROR_FILE

Reply:
Sounds good. Originally, I thought --error is consistent with --root and --dest, but both of --root and --dest are directories and --error seems a little bit vague.
DONE